src/guiguts/highlight.py

```diff
@@ -88,6 +90,12 @@ class HighlightColors:
         "Default": {"bg": "dodgerblue", "fg": "white"},
     }
 
+    ALIGNCOL = {
+        "Light": {"bg": "greenyellow", "fg": "black"},
+        "Dark": {"bg": "#577a32", "fg": "white"},
+        "Default": {"bg": "greenyellow", "fg": "black"},
```

Comment:
Actually, there is, and that's what I did in the next PR (#528). I kind of like it - it adapts dynamically if the theme changes (although you have to do something to trigger the after_idle, which is almost anything).

@windymilla if you look at #528 (the second commit), what if we did something like that and moved it down the stack into `_highlight_configure_tag()`? That way it would adapt to dark vs. light based on `maintext().is_dark_theme()` and we'd only have to define a Light and a Dark color set, and skip Default.

Reply:
I agree that it would be probably be better to do that (make everything adapt to `is_dark_theme` rather than the theme name). I think it would better to get all the highlighting PRs merged first though.

Reply:
In Default theme, with macOS dark mode on, it's hard to control the color without refactoring to use `is_dark_theme()` - I agree with Nigel about merging these and coming back to clean that up, and we have another issue open already to revisit the color situation. So I'll leave this as-is for now, and we'll clean up the colors later.